package.json

```diff
@@ -36,7 +39,13 @@
     "example": "examples"
   },
   "dependencies": {
-    "jsx-md": "^1.2.0"
+    "chalk": "^4.1.0",
+    "clear": "^0.1.0",
+    "commander": "^6.1.0",
+    "figlet": "^1.5.0",
+    "fs-extra": "^9.0.1",
+    "jsx-md": "^1.2.0",
+    "path": "^0.12.7"
```

Comment:
Do we need all of these? Specifically `path` is already built into node. I would prefer to have as few dependencies as possible.

Reply:
figlet and fs-extra will be removed, that would leave us only with path if it's ok with you

Reply:
`path` is a built-in module, there is no need to install it from npm.